definitions/.redocly.yaml

```diff
@@ -0,0 +1,64 @@
+apiDefinitions:
+  sample: ./syntetic/to_bundle/bundle.yaml
+  simpleDefinition: ./syntetic/syntetic.yaml
+  rebilly: ./openapi-directory/rebilly-full.yaml
+
+lint:
+  codeframes: on
+  headers:
+    - name: Client-Header
+      value: sometestvalue
+      regexp: api.redoc.ly/.*
```

Comment:
Regexp may be confusing to some users and not always required, could we add `hostname` syntax sugar ?

Also this specific regexp is insecure, see bellow for details.